libs/audit-service/src/main/java/uk/gov/di/ipv/core/library/auditing/AuditEvent.java

```diff
@@ -35,7 +36,7 @@ public class AuditEvent {
     private final AuditRestricted restricted;
 
     @JsonCreator
-    public AuditEvent(
+    private AuditEvent(
```

Comment:
exactly - and you could remove/privatise the remaining constructors as well (but that'll require refactoring all the existing audit events)